apps/framework-cli/src/infrastructure/olap/clickhouse/inserter.rs

```diff
@@ -140,8 +141,10 @@ impl<C: ClickHouseClientTrait + 'static> Inserter<C> {
                         }
                     }
                     Err(e) => {
-                        error!("Error inserting records to {}: {:?}", table, e);
-                        debug!("Failed batch size: {}", batch_size);
+                        error!(
+                            "Error inserting records {} to {}: {:?}",
+                            batch_size, table, e
```

Comment:
The `batch_size` `{}` should go before "records"?

Reply:
English 😏